python/orca/tutorial/NCF/pytorch_resume_train_dataloader.py

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Step 0: Import necessary libraries
+import torch.nn as nn
+import torch.optim as optim
+import torch.utils.data as data
+
+from pytorch_dataset import load_dataset, process_users_items, get_input_dims
+from pytorch_model import NCF
+from utils import *
+
+from bigdl.orca.learn.pytorch import Estimator
+from bigdl.orca.learn.pytorch.callbacks.tensorboard import TensorBoardCallback
+from bigdl.orca.learn.metrics import Accuracy, Precision, Recall
+
+
+# Step 1: Init Orca Context
+args = parse_args("PyTorch NCF Resume Training with DataLoader")
+init_orca(args.cluster_mode, extra_python_lib="pytorch_model.py,pytorch_dataset.py")
+
+
+# Step 2: Define train and test datasets as PyTorch DataLoader
+def train_loader_func(config, batch_size):
+    train_dataset, _ = load_dataset(config["data_dir"], num_ng=4)

```

Comment:
Seems this should be `config["dataset_dir"]`? It's not consist with https://github.com/intel-analytics/BigDL/blob/main/python/orca/tutorial/NCF/pytorch_train_dataloader.py#L64

Here's what's in my local `config.json`
```json
{"dataset_dir": "./ml-1m", "factor_num": 16, "num_layers": 3, "dropout": 0.5, "lr": 0.01, "sparse_feats_embed_dims": 8}
```